bot/prisma/schema.prisma

```diff
@@ -286,6 +289,20 @@ model Birthday {
   @@unique([userId, guildId])
 }
 
+// Reminders
+model Reminder {
+  id        String   @id @default(cuid())
+  createdAt DateTime @default(now())
+
+  userId     String
```

Comment:
i'd group messageUrl, description and triggersAt together and user and userId as well
```prisma
  messageUrl String?
  description String?
  triggersAt  DateTime
  
  userId String
  user  User  @relation(fields: [userId], references: [id], onDelete: Cascade, onUpdate: Cascade)
  ```